s2a_options.go

```diff
@@ -123,11 +136,16 @@ type ServerOptions struct {
 	S2AAddress string
 	// If true, enables the use of S2Av2.
 	EnableV2 bool
+	// VerificationMode specifies the verification mode supported by S2Av2.
```

Comment:
Suggested edit: VerificationMode specifies the mode that S2A must use to verify the peer certificate chain.

Reply:
Will do, thanks!